### ISSUE TITLE
refer to steps defined in a different suite/testCase

### ISSUE THREAD

**Issue by mox601** [HotelsDotCom/heat#33]
in `e2eflowSteps` can we refer to steps defined in a different suite/testCase? 
this prevents copy/pasting steps around, and improves code reuse.

**Comment by lucap84** [comment]
No, you can only refer to other steps in the same flow: [see docs here](https://github.com/HotelsDotCom/heat/blob/master/doc/readme_flow.md#pass-information-among-steps)
